Sources/HaCWebsiteLib/ViewModels/PostCard.swift

```diff
@@ -0,0 +1,32 @@
+import HaCTML
+
+struct PostCard {
+  enum PostCategory: String {
+    case workshop = "Workshop"
+    case video = "Video"
+    case hackathon = "Hackathon"
+  }
+
+  let title: String
+  let category: PostCategory
+  let description: String
+  let backgroundColor: String?
+  let imageURL: String?
+  
+  var node: Node {
```

Comment:
This is awesome!

Reply:
Once we have made a few more components and have played with different patterns we can think about things like `ViewModel` protocols etc if we want to